pages/domains/[name]/index.tsx

```diff
@@ -51,6 +52,7 @@ export function ManageNamePageContent({ name }: { name: string }) {
     handleFetchENSDomainData();
   }, [name]);
 
+  // If the domain is not available, redirect to the register page
```

Comment:
Do you find another word to express "available"? It doesn't fit in here for me since a user should only be sent to register page if a name is available

Reply:
The idea was if a domain data is available to be fetched by the domain page. I'll update the comment 😄